### PR TITLE
#151168818 Delete all notifications

### PR DESCRIPTION
**Pull request by dennomwas**
### What does this PR do?

Feature to delete  notifications from redis

### Description of Tasks completed
- Add delete all notifications endpoint
- Add delete notification route
- Write associated tests

#### How should this be manually tested? 

After cloning the repo cd into it and run;
```
pip install -r requirements.txt
```

#### To run the tests
```
DB_TYPE=sqlite pytest
```
#### What are the relevant pivotal tracker stories? ####

[Delivers #151168818]

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -121,6 +121,8 @@ def index():
                      '/api/v1/notifications/',
                      '/api/v1/notifications/<string:notif_id>',
                      '/api/v1/notifications/<string:notif_id>/',
+                     '/api/v1/notifications/delete/<string:user_id>/',
```
Comment (Mnickii) [inline main.py]: We already have the endpoints so this is not necessary. We are using one URL for every CRUD action. @dennomwas

File: main.py [hunk 2]
```diff
@@ -121,6 +121,8 @@ def index():
...
+                     '/api/v1/notifications/delete/<string:user_id>',
```
Comment (Mnickii) [inline main.py]: This too

Comment (dennomwas) [reply main.py]: Okay let me refactor this.... thanks

**Comment by Mnickii** [comment]
LGTM

**Comment by andela-ojoloko** [comment]
LGTM

**Comment by andela-anandaa** [comment]
I think notifications are like logs, they should never be deleted...

**Pull request closed without merging**